### PR TITLE
[postgres] update end batch time for Postgres destinations

### PR DESCRIPTION
**Pull request by heavycrystal**
looks like it was removed from flowable at some point

### CODE REVIEW

File: flow/activities/flowable.go [hunk 1]
```diff
@@ -361,6 +361,17 @@ func (a *FlowableActivity) StartNormalize(
 		a.Alerter.LogFlowError(ctx, input.FlowConnectionConfigs.FlowJobName, err)
 		return nil, fmt.Errorf("failed to normalized records: %w", err)
 	}
+	dstType, err := connectors.LoadPeerType(ctx, a.CatalogPool, input.FlowConnectionConfigs.DestinationName)
+	if err != nil {
+		return nil, err
+	}
+	if dstType == protos.DBType_POSTGRES {
```
[the remark this answers is not in the document]
Comment (heavycrystal) [reply flow/activities/flowable.go:368]: `UpdateNormalizeBatchID` is only available to connectors who use catalog to store sync information (`PostgresMetadata`)
Postgres doesn't do that

[the remark this answers is not in the document]
Comment (iskakaushik) [reply flow/activities/flowable.go:368]: can you remind me, where does PG store its state?

[the remark this answers is not in the document]
Comment (heavycrystal) [reply flow/activities/flowable.go:368]: tables are created under `metadataSchema` on dst, defaults to `_peerdb_internal`

**Review by iskakaushik — APPROVED** [review]

**Comment by serprex** [comment]
removed by https://github.com/PeerDB-io/peerdb/pull/2076 where we assumed every connector used external metadata store

**Pull request merged**